src/stylesheets/layouts/_factors.scss

```diff
@@ -146,8 +141,7 @@
     border: 5px solid $color-white;
     box-shadow: 0 0 0 5px rgba(#ffffff, 0.6);
     left: 0;
-    margin-left: auto;
-    margin-right: auto;
+    margin: auto;
```

Comment:
Are you safe setting margin top/bottom to `auto` as well? It's probably fine, but `margin: 0 auto;` is generally the way to be explicit (and safe) about margins so you know what you're getting.